integration/tests/api/docker_test.go

```diff
@@ -211,7 +212,15 @@ func TestDockerContainerSpec(t *testing.T) {
 	assert := assert.New(t)
 
 	assert.True(containerInfo.Spec.HasCpu, "CPU should be isolated")
-	assert.Equal(cpuShares, containerInfo.Spec.Cpu.Limit, "Container should have %d shares, has %d", cpuShares, containerInfo.Spec.Cpu.Limit)
+	if cgroups.IsCgroup2UnifiedMode() {
+		// see https://github.com/google/cadvisor/blob/24e7a9883d12f944fd4403861707f4bafcaf4f3d/container/common/helpers.go#L249-L260
```

Comment:
Could you add more context to this comment. Is it pointing out how shares are calculated?

If so, perhaps replace `// see` with `// see how cpu shares are calculated`

Reply:
updated comment

Reply:
![](https://i.kym-cdn.com/news/images/desktop/000/000/142/faf.jpg)

What a beautiful comment :D, thanks.